five.js

```diff
@@ -44,6 +44,7 @@
   five.indonesian = function() { return 'lima'; };
   five.irish = function() { return 'cúig'; };
   five.italian = function() { return 'cinque'; };
+  five.icelandic = function() { return 'fimm'; };
```

Comment:
Can you move this up above `indonesian` so that it maintains alphabetical order? :) same in the test file and readme